src/fillTank.test.js

```diff
@@ -1,11 +1,107 @@
-'use strict';
+"use strict";
 
-describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+describe("fillTank", () => {
+  const { fillTank } = require("./fillTank");
 
-  it('should ', () => {
+  it(`should be full tank because 'amount' was not passed`, () => {
+    const customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 8,
+      },
+    };
 
+    fillTank(customer, 1);
+
+    expect(customer.vehicle.fuelRemains).toBe(40);
+  });
+
+  it(`should be a full tank
+      if the 'amount' is larger than the tank capacity`, () => {
+    const customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 8,
+      },
+    };
+
+    fillTank(customer, 1, 45);
+
+    expect(customer.vehicle.fuelRemains).toBe(40);
+  });
+
+  it(`should update 'fuelRemains'
+      with the 'amount' of fuel the customer can pay for`, () => {
+    const customer = {
+      money: 100,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 10,
+      },
+    };
+
+    fillTank(customer, 11, 10);
```

Comment:
The test case assumes the customer can pay for 9 liters (100 money / 11 fuelPrice), but the test expects 19 liters in the tank. Ensure the calculation matches the expected result.